src/Features/Core/Portable/CodeLens/CodeLensFindReferenceProgress.cs

```diff
@@ -58,14 +58,6 @@ public void OnCompleted()
     {
     }
 
-    public void OnFindInDocumentStarted(Document document)
```

Comment:
FAR had a system to notify consumers "i'm startign a doc" and "i'm done with a doc".  The original thinking was to use that for progress, to display which file we're looking at.

but:

1. we never hooked this up to naything at all.
2. this wouldn't be a good experience anyways, since we're extremely parallel. so this was removed entirely.